### ISSUE TITLE
The label of Imagenet 22k

### ISSUE THREAD

**Issue by sweetorangezhuyan** [microsoft/CSWin-Transformer#29]
Could you provide the corresponding labels for Imagenet 22k? Looking forward to your reply. Thank you !

**Comment by kaierlong** [comment]
@sweetorangezhuyan Hi, Did you get the labels file for Imagenet 22k?

**Comment by sweetorangezhuyan** [comment]
Yeah, this is the labels file provided by the author of CSWin-Transformer.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "microsoft/CSWin-Transformer"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年6月22日(星期三) 上午10:13
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [microsoft/CSWin-Transformer] The label of Imagenet 22k (Issue #29)





 
@sweetorangezhuyan Hi, Did you get the labels file for Imagenet 22k?

**Comment by kaierlong** [comment]
@sweetorangezhuyan 
Sorry, I didn't find any file from your reply.
By the way, did you get the 22kto1k mapping file right. 
If so, can you send that file to my email (kaierlong@126.com)?
